contracts/levels/Fallback.sol

```diff
@@ -1,22 +1,35 @@
 pragma solidity ^0.5.0;
 
-import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
 import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
 
-contract Fallback is Ownable {
+contract Fallback {
 
   using SafeMath for uint256;
   mapping(address => uint) public contributions;
+  address payable _owner;
 
   constructor() public {
+    _owner = msg.sender;
     contributions[msg.sender] = 1000 * (1 ether);
   }
 
+   function owner() public view returns (address payable) {
+        return _owner;
+    }
+
+  modifier onlyOwner {
+        require(
+            msg.sender == _owner,
+            "Only owner can call this function."
```

Comment:
It might be a good idea to use an error message consistent with the one used in OpenZeppelin:

```suggestion
            "caller is not the owner"
```